01-davinci.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: First Steps in LHCb
+subtitle: Introduction to the LHCb Software
+minutes: 10
+---
+
+> ## Learning Objectives {.objectives}
+> * Which basic problems is the LHCb software framework designed to solve?
+> * Learn the key concepts needed to work with the LHCb software
+>
+
+A HEP experiment like LHCb needs to solve many computational challenges, some of which are
+
+ - How do we collect data as it is recorded by the detector?
+ - How do we filter and process the recorded data efficiently?
+ - How do we manage all the complex tasks required to work with particle events?
```

Comment:
I would rather avoid going into a definition of particle events here.
Maybe *collision data* would be more appropriate?
This line is supposed to be the motivation for having `Tools` in Gaudi.